README.md

```diff
@@ -1,17 +1,10 @@
 # scarpet for VSCode
 
-[![vscode version](https://img.shields.io/visual-studio-marketplace/v/ImUrX.scarpet)](https://marketplace.visualstudio.com/items?itemName=ImUrX.scarpet)
+## How to use
```

Comment:
you just deleted what i did and copy+pasted it but dw, do it with this too

Reply:
ah, I thought that was an accident

Reply:
but wait, what do u want me to do?

Reply:
done